website/docs/cloud-docs/policy-enforcement/sentinel/import/tfrun.mdx

```diff
@@ -11,6 +11,11 @@ The `tfrun` import provides access to data associated with a [Terraform run][run
 @include 'tfc-package-callouts/policies.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
+<!-- BEGIN: TFC:only name:pnp-callout -->
+The Import will exclude `cost_estimate` data if using Enhanced Policy evaluation See [Policy Evaluation](terraform/cloud-docs/policy-enforcement/manage-policy-sets#policy-evaluations) for more details.
```

Comment:
Do we still need to inform folks about TF plan details here? If so, maybe we can leave the callout as is, and add this sentence later on?

I could see this sentence fitting nicely in the paragraph below after the first sentence, like so:
 "This import currently consists of run attributes, as well as namespaces for the `organization`, `workspace` and `cost-estimate`. Note that imports exclude `cost_estimate` data if you are using enhanced policy evaluations. See [Policy Evaluation](/terraform/cloud-docs/policy-enforcement/manage-policy-sets#policy-evaluations) for more details."

Does that feel good to you? Also, please let me know if what I'm saying doesn't make sense 😂